framework/widgets/Pjax.php

```diff
@@ -99,13 +99,13 @@ class Pjax extends Widget
     public $clientOptions;
     /**
      * @inheritdoc
-     * Note: This variable is redeclared to force separate counting for Pjax widgets (issue #12969)
-     * @since 2.0.11
+     * @internal
+     * @since 2.0.11 (#12969)
```

Comment:
> doesnt it matter that the counter is reset to 0 for Pjax Widgets?

It matters but it's not API change and `@since` (poor guy, he's mentioned all the time...) is for public API changes and in this case there aren't any.